protocol/localstatequery/client.go

```diff
@@ -499,6 +500,12 @@ func (c *Client) GetUTxOWhole() (*UTxOWholeResult, error) {
 	if err := c.runQuery(query, &result); err != nil {
 		return nil, err
 	}
+	jsonData, err := json.MarshalIndent(result, "", "  ")
+	if err != nil {
+		return nil, fmt.Errorf("error marshaling UTxOWhole result to JSON: %w", err)
+	}
+	fmt.Println("UTxOWhole Result:")
+	fmt.Println(string(jsonData))
```

Comment:
Remove this. We only want changes (currently) in the binary so we can determine the correct shape.